packages/nextjs-kit/src/components/footer.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import Link from 'next/link';
+import { Menu } from '../index';
+
+interface MenuProps {
+  menuItems: Menu[];
+  children: JSX.Element;
+}
+
+/**
+ * This is a Footer component.
+ *
+ * @param {Props} menuItems holds the title, href, and id of each route to be displayed
+ * @param {Props} children  copyright display and information to be passed in by the user
+ * @returns {React.JSX.Element}
+ */
+const Footer: React.FC<MenuProps> = ({ menuItems, children }: MenuProps) => {
+  const FooterMenu = () => (
+    <nav className="flex flex-col max-w-lg mx-auto lg:max-w-screen-lg">
+      <ul>
+        {menuItems.map(({ label, path, id }) => {
+          return (
+            <li
+              key={id}
+              className="list-disc text-blue-300 hover:text-blue-100 ml-3"
+            >
+              <Link href={`/posts${path}`}>
```

Comment:
```suggestion
              <Link href={`/${path}`}>
```
I thought this was only in the snapshot component before so I didn't catch this.
You may also need to check for leading `/` on the path and remove it to avoid passing in `//path`.